opm/parser/eclipse/EclipseState/SummaryConfig/tests/SummaryConfigTests.cpp

```diff
@@ -55,8 +55,9 @@ static DeckPtr createDeck( const std::string& summary ) {
             "/\n"
             "COMPDAT\n"
             "'PRODUCER'   5  5  1  1 'OPEN' 1* -1  0.5  / \n"
-            "'W_1'   3    7    1    3      'OPEN'  1*     32.948      0.311   3047.839  2*         'X'     22.100 / \n"
             "'W_1'   3    7    2    2      'OPEN'  1*          *      0.311   4332.346  2*         'X'     22.123 / \n"
```

Comment:
Correct in Eclipse you have to enter the form n* (1*, 2* etc.) * on its 
own will throw a warning/error.

ThunderBird Signature File

Regards, David Baxendale Email: David.Baxendale@gmail.com

------------------------------------------------------------------------
On 15/07/2016 20:03, Kai Bao wrote:
>
> In 
> opm/parser/eclipse/EclipseState/SummaryConfig/tests/SummaryConfigTests.cpp 
> <https://github.com/OPM/opm-parser/pull/881#discussion_r70961402>:
>
> > @@ -55,8 +55,9 @@ static DeckPtr createDeck( const std::string& summary ) {
> >              "/\n"
> >              "COMPDAT\n"
> >              "'PRODUCER'   5  5  1  1 'OPEN' 1* -1  0.5  / \n"
> > -            "'W_1'   3    7    1    3      'OPEN'  1*     32.948      0.311   3047.839  2*         'X'     22.100 / \n"
> >              "'W_1'   3    7    2    2      'OPEN'  1*          *      0.311   4332.346  2*         'X'     22.123 / \n"
>
> I mean here. I just realized that you deleted the above line while 
> this line is not added by you. I misread it. Sorry about that.
>
> |*| indicating defaulted values are not supported by Eclipse, while we 
> have so many this kinds of examples in our data decks in the tests.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/OPM/opm-parser/pull/881/files/7a682fb691fee11a06bff2921370c180c9bd8036#r70961402>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/ARIID3A3GaxFuAGsXJBIk50FTulQ-Dfqks5qV3cQgaJpZM4JNT3o>.
>